terraform/modules/indexer/rds.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_db_subnet_group" "indexer" {
+  name       = "indexer"
+  subnet_ids = var.subnet_ids
+
+  tags = local.default_tags
+}
+
+resource "aws_security_group" "indexer" {
+  name   = "indexer-${local.workspace_name}"
+  vpc_id = var.vpc_id
+  tags   = local.default_tags
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+
+resource "aws_db_instance" "indexer" {
+  identifier             = "indexer-${local.workspace_name}"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
+  engine                 = "postgres"
+  engine_version         = "14.1"
+  username               = "indexer"
+  password               = var.db_password
+  db_subnet_group_name   = aws_db_subnet_group.indexer.name
+  vpc_security_group_ids = [aws_security_group.indexer.id]
+  parameter_group_name   = aws_db_parameter_group.indexer.name
+  publicly_accessible    = false
+  skip_final_snapshot    = true
+}
+
+resource "aws_db_parameter_group" "indexer" {
+  name   = "indexer"
+  # family parameter must correspond with the enginve version of aws_db_instance.indexer
+  family = "postgres14"
```

Comment:
could also make the engine version configurable, and derive this value from the version if they need to match.

Reply:
I'll make it configurable. It's non-obvious to derive since sometimes the minor versions can be different. I'll just keep the note